content/guides/tasks/assignments/assign.md

```diff
@@ -0,0 +1,15 @@
+---
+rank: 2
+related_endpoints:
+  - post_task_assignments
+related_guides:
+required_guides: []
+alias_paths: []
+---
+
+# Assign a task to a user
+
+To assign a task to a user in Box, you will need to provide our API with a task
+`id` and a user `id` to assign it to.
+
+<Samples id='post_task_assignments' />
```

Comment:
There are a few variations here. Can we cover these? Don't need to add them as samples, the simplest things would be to add a section for the obvious use cases and then a quick sample. In this case I'd like to cover: assigning to a user by login.